.github/workflows/ci.yml

```diff
@@ -65,9 +65,32 @@ jobs:
         scandir: './crypto'
         ignore: 'relic'
 
+  create-dynamic-test-matrix:
+    name: Create Dynamic Test Matrix
+    runs-on: ubuntu-latest
+    outputs:
+      dynamic-matrix: ${{ steps.set-test-matrix.outputs.dynamicMatrix }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          cache: true
+      - name: Set Test Matrix
+        id: set-test-matrix
+        run: go run utils/test_matrix/test_matrix.go network engine access admin utils
+
   unit-test:
     name: Unit Tests
```

Comment:
```suggestion
    name: Unit Tests (${{ matrix.targets.name }})
```

Reply:
added in https://github.com/onflow/flow-go/pull/3087/commits/635fbd32ef269e53491cecce5c373fa809f5a83e